modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1020,3 +1020,9 @@
 		margin-top: 20px;
 	}
 }
+
```

Comment:
Hey @ramonmsp, Can you add some specific calendar class to make the selector specific for the calendar portlet?

Just to make sure that this change will only affect the calendar portlet when we use the it together with others portlets on the same page

Thx

Reply:
I added one more class (calendar-portlet). It's enought?

Reply:
yeap